api/routes/profile_page.py

```diff
@@ -114,4 +115,28 @@ def update_user_status(user_id: int, new_status: UserStatus, db: Session = Depen
 
     user.status = new_status
     db.commit()
-    return user
\ No newline at end of file
+    return user
+
+@profile_route.get("/username")
```

Comment:
I don't think it is good practice to use a get method for making changes to the db. You can make it a post request with an empty payload.